examples/plot_simulate_evoked.py

```diff
@@ -81,18 +127,25 @@
 print('Mean spike rates for individual trials:')
 print(trial_rates)
 
-
 ###############################################################################
 # Now, let us try to make the exogenous driving inputs to the cells
-# synchronous and see what happens
+# synchronous and see what happens. This is achieved by setting ``sigma=0```.
+# Using the ``copy``-method, we can create a clone of the network defined
+# above. Then modify the drive dynamics for each drive.
+
+net_sync = net.copy()
+net_sync.external_drives['evdist1']['dynamics']['sigma'] = 0
+net_sync.external_drives['evprox1']['dynamics']['sigma'] = 0
+net_sync.external_drives['evprox2']['dynamics']['sigma'] = 0
```

Comment:
Setting sigma to 0 will also synchronize the evoked input times across trials right? If so, I think the goal of a "synchronous" input is to synchronize timing across cells within a trial while allowing for jitter across trials.

Reply:
Ah, finally I get `sync_ev`... :( Bugger, I was so happy to see that parameter go!

Could we avoid it? By e.g. adding a new param such as `inter_trial_sigma`? I know it ain't pretty, but `sync_ev` isn't descriptive either, is it?

Reply:
Took a stab at it in 6bf4a6c. It now reproduces the docs for a single trial, and for `n_trials=2`, it produces non-identical traces:

<img width="514" alt="Screenshot 2021-01-05 at 12 28 13" src="https://user-images.githubusercontent.com/5610076/103641818-9de6e700-4f52-11eb-80c3-772f779f76b5.png">

Good to go?

Reply:
Hmm, I can't get your branch to produce that plot. Did something break in https://github.com/jonescompneurolab/hnn-core/pull/221/commits/aaeba250b65bbf2562e2398bd7e211a1fc5532da?

Reply:
Sorry, I wasn't explicit about the syntax, still works as above. You no longer set `sigma=0`, but rather (see `plot_simulate_evoked.py`):

```python
net_sync = net.copy()
net_sync.external_drives['evdist1']['dynamics']['sync_within_trial'] = True
net_sync.external_drives['evprox1']['dynamics']['sync_within_trial'] = True
net_sync.external_drives['evprox2']['dynamics']['sync_within_trial'] = True
```

Then get the `dpls_sync` and `plot_dipole` to get the above plot. I checked the spike time of the artificial cells in the two trials:

```python
evp1: trial0=26.734887416643062,trial1=27.699831892606053
evd1: trial0=63.72466257250032,trial1=65.2287258245074
evp2: trial0=137.5411790205007,trial1=140.7954249657524
```

Subtly different, as determined by the `sigma`-values of each.

Reply:
I'm running `plot_simulte_evoked.py` on aaeba25 and it doesn't produce a different plot after setting `sync_within_trial` to `True`.